components/tag-list/tag-list-item-mixin.js

```diff
@@ -200,13 +203,16 @@ export const TagListItemMixin = superclass => class extends LocalizeCoreElement(
 		));
 	}
 
-	_renderTag(tagContent, hasTruncationTooltip) {
+	_renderTag(tagContent, hasTruncationTooltip, description) {
 		const buttonText = typeof tagContent === 'object'
 			? this.localize('components.tag-list.clear', { value: '' })
 			: this.localize('components.tag-list.clear', { value: tagContent });
+		const hasDescription = !!description;
+		const tooltipHeader = hasDescription ? html`<div class="d2l-heading-4">${tagContent}</div>` : tagContent;
```

Comment:
@dbatiste using a `div` here to get a line break and adjusted margin to be more reasonable. See before/after margin changes:
| Default `margin-bottom` | Adjusted Margin | 
| --- | --- |
| ![tag-list-before](https://user-images.githubusercontent.com/64804046/185428653-91467ab1-e55c-4473-921a-269af4c5a5a7.PNG) | ![tag-list-after](https://user-images.githubusercontent.com/64804046/185428682-9338a414-d7d2-46a5-a5d5-e44806a9bc44.PNG) |

Reply:
I think it should be good.  I will test it out just to make sure.

Reply:
So it doesn't announce when the `div` is used either. I also tried a `span` with `display: block` and it does't get announced either. Need to play around with it more.

Reply:
@dbatiste saw the conversation on slack regarding this issue, given that there is no immediate solution can this be merged as is since there are existing accessibility issues with this component? (see [resolved comment here](https://github.com/BrightspaceUI/core/pull/2772#discussion_r948250198) regarding the existing issue)

Reply:
Yeah I think so.